### DISCUSSION TITLE
Is there a way to have appended properties also be transformed by global transformers?

### DISCUSSION THREAD

**Discussion by galexy** [spatie/laravel-data#452] [Q&A]
I have a Data class that uses `with` to append properties. One of these properties is a Carbon datetime. However, this datetime is not being transformed with the formatting option `date_format` in my `data.php`. Is there a way to workaround this and is this a known issue?

**Comment by rubenvanassche** [comment]
Everything added to with doesn't go through the transformation process, you'll need to add native properties if you want this.
